cli.js

```diff
@@ -1968,6 +1968,53 @@ program
     });
 
 
+program
+    .command('user:reset')
+    .description('Reset a user')
+    .option('-l, --login <login>', 'Login of the user to reset')
+    .option('-j, --json', 'Formats the output in json')
+    .option('-N, --noprompt', 'No prompt to confirm reset')
+    .action(function (options) {
+        var login = options.login;
+        var asJson = (options.json ? options.json : false);
+        var noPrompt = (options.noprompt ? options.noprompt : false);
+
+        var resetUser = function (login, asJson) {
+            require('./lib/user').cli.reset(login, asJson);
+        };
+
+        if (!login) {
+            require('./lib/log').error('Missing required --login. Use -h,--help for help.');
+        } else if (noPrompt) {
+            resetUser(login, asJson);
+        } else {
+            prompt({
+                type: 'confirm',
+                name: 'ok',
+                default: false,
+                message: 'Reset user ' + login +
+                    '. Are you sure?'
+            }).then((answers) => {
+                if (answers['ok']) {
+                    resetUser(login, asJson);
+                }
+            });
+        }
+    }).on('--help', function () {
+        console.log('');
+        console.log('  Details:');
+        console.log();
+        console.log('  Reset a user.');
```

Comment:
Can you please add some more details here on what the rest does? My suggestion:

> Reset a user. This is useful when the user forgot their password or when you want to unlink the Account Manager user account from a Salesforce account. An email is automatically sent to the user's email address with an activation link.
> This requires permissions ...

Reply:
Added @tobiaslohr